pkg/resourceinterpreter/defaultinterpreter/prune/prune.go

```diff
@@ -80,7 +81,10 @@ func RemoveIrrelevantField(workload *unstructured.Unstructured) error {
 			_ = unstructured.SetNestedSlice(workload.Object, secrets, "secrets")
 		}
 	}
-
+	// removes to specify nested field from the obj
```

Comment:
Just one question, I don't understand the meaning of the comment.

Reply:
Remove from object to specify nested fields

Reply:
What comments do you think should be written here?

Reply:
I think we can just remove it as it's straightforward enough, or add a comment like 
```
// removes extra fields addressed by hooks.
```

Reply:
>  just remove it 
ok